src/modules/checkIsValidUserInput.js

```diff
@@ -9,7 +9,21 @@
  * @return {boolean} - True if the user input is valid, false otherwise
  */
 function checkIsValidUserInput(userInput) {
-  /* Write your code here */
+  if (userInput.length !== 4 || isNaN(Number(userInput))) {
```

Comment:
Magic number. What mean 4? 
Move this to constant and set name what mean this number.